packages/core/src/components/dialog/dialog.tsx

```diff
@@ -115,7 +115,7 @@ export class Dialog extends AbstractPureComponent<IDialogProps, {}> {
         return (
             <div className={Classes.DIALOG_HEADER}>
                 <Icon icon={icon} iconSize={Icon.SIZE_LARGE} />
-                <h5>{title}</h5>
+                <h4 className={Classes.DIALOG_HEADER_TITLE}>{title}</h4>
```

Comment:
gotta merge 1.x into develop (and then into this branch) for `Classes` constant to exist